baselines/commoncontrols.md

```diff
@@ -449,6 +461,8 @@ User passwords SHALL NOT expire.
 
 -   [GWS Admin Help \| Enforce and monitor password requirements for users](https://support.google.com/a/answer/139399?hl=en#zippy=%2Cwhat-makes-a-password-strong)
 -   [CIS Google Workspace Foundations Benchmark](https://www.cisecurity.org/benchmark/google_workspace)
```

Comment:
For what it is worth, our baseline has diverged enough from CIS guidance that it might not make sense to include this as a resource anymore:
- CIS mandates password expiration (1 year)
- CIS recommends 14 as the minimum length